src/baldr/graphtile.cc

```diff
@@ -449,12 +449,14 @@ std::string GraphTile::FileSuffix(const GraphId& graphid,
                                    : TileHierarchy::levels()[graphid.level()]);
 
   // figure out how many digits in tile-id
-  const auto max_id = level.tiles.ncolumns() * level.tiles.nrows() - 1;
+  const uint32_t max_id = level.tiles.ncolumns() > 0 && level.tiles.nrows() > 0
+                              ? level.tiles.ncolumns() * level.tiles.nrows() - 1
+                              : 0;
```

Comment:
this is not needed we do not allow for negative numbers in the dimensions of the tiles datastructure. in fact i forget why we even use signed integers in that class. at any rate, to avoid the warning use a `static_cast`

Reply:
Its not negative number on `ncolumns` or `nrows` the issue here,
its what happens when any of those is 0, the max_id gets a -1, but its a uint32_t.

I will change to `static_cast`.